src/components/About/About.js

```diff
@@ -13,29 +14,41 @@ class About extends React.Component {
         repoList: [],
         userData: [],
         isError: false,
-        errorMessage: ""
+        errorMessage: "",
+        currentPage: 1,
+        perPage: 4
     }
 
     componentDidMount() {
-        octokit.repos.listForUser({
+        this.requestRepolist(this.state.perPage, this.state.currentPage);
+
+        octokit.users.getByUsername({
             username: "karolinaewagorska"
+          }).then(({data}) => {
+              this.setState({
+                  userData: data,
+                  isLoading: false});
+          }).catch((error) => {
+              this.setState({
+                  isLoading: false,
+                  isError: true,
+                  errorMessage: error
+            });
+        });   
+    }
+
+    requestRepolist = (perPage, selectedPage) => {
+        octokit.repos.listForUser({
+            username: "karolinaewagorska",
+            per_page: perPage,
```

Comment:
Codacy found an issue: [Identifier 'per_page' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)